src/mono/wasm/runtime/cjs/dotnet.cjs.extpost.js

```diff
@@ -1,3 +1,4 @@
+var require = require || ((name) => { return Module.imports.require(name) });
```

Comment:
when will `|| ((name) => { return Module.imports.require(name) })` ever be true in cjs?

Reply:
when somebody wanted to replace by setting `imports.require`. This is for consistency.

Reply:
I see, thanks!

Reply:
You mean that in csj require would be always defined. But we are also loading cjs-shaped dotnet.js as backward compatibility into global namespace.  In such case require may not be there. It will be replaced during `initializeImportsAndExports` anyway.